packages/opentelemetry-instrumentation-sagemaker/opentelemetry/instrumentation/sagemaker/__init__.py

```diff
@@ -145,16 +145,31 @@ def stream_done(response_body):
     response["Body"] = StreamingWrapper(response["Body"], stream_done)
 
 
+def _try_parse_json(value):
```

Comment:
I think a more pythonic way here is to use `json.loads` and pass a custom decoder, wdyt?